### DISCUSSION TITLE
Wanting to use the heart rate input, what is the best hardware and software to get?

### DISCUSSION THREAD

**Discussion by WolfInside** [vrchat-community/osc#79] [Q&A]
I am completely unknowledgable when it comes to this topic of hardware/software implementation for the OSC implementation. How would I go about learning this tech and are there any suggestions for what I should use?

Edit: I forgot to mention I have some experience involving coding (Py, C++)

**Comment by Janni9009** [comment]
There's 200TigersBloxed's [HRtoVRChat_OSC](https://github.com/200Tigersbloxed/HRtoVRChat_OSC) which has great documentation for anything software and avatar side, and anything that supports the senders it lists will be compatible. Personally I've seen a few people use Pine64's PineTime, which I'm gonna be ordering as well the coming month since it seems quite reliable and cheap at that, and some using fitbits and samsung watches.
